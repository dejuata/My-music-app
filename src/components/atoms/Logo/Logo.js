import React from 'react'


export const Logo = () => {

    const logoPath = `./assets/logo.svg`;

    return (
        <div className='a-logo-container'>
            <div className='a-logo-container__image'>
                <img src={logoPath} alt="logo" />
            </div>
            <p className='a-logo-container__title'>MY MUSIC APP</p>
        </div>
    )
}
