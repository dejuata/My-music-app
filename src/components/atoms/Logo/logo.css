.a-logo-container {
    width: 100%;
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
    margin-bottom: 20px;
}

.a-logo-container__image {
    width: 120px;
    height: 120px;
    background-color: var(--color-white);
    border-radius: 50%;
    display: flex;
    justify-content: center;
    align-items: center;

}

.a-logo-container__image img {
    width: 90px;
    height: 90px;
    padding: 2px;
    box-sizing: border-box;
    min-width: 40px;
    min-height: 40px;
}

.a-logo-container__title {
    margin: 1rem 0 0;
    font-weight: bold;
    font-size: var(--xlg);
}

/* body {
    background-color: var(--color-dark-white);
} */