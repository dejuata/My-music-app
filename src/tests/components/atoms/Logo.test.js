import React from 'react';
import { shallow } from 'enzyme';
import { Logo } from '../../../components/atoms/Logo/Logo';

describe('Test Logo', () => {

    const wrapper = shallow(
        <Logo />
    )


    test('Should be render Logo', () => {

        expect(wrapper).toMatchSnapshot();

    })

})