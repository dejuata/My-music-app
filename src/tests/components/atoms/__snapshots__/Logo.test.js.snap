// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Logo Should be render Logo 1`] = `
<div
  className="a-logo-container"
>
  <div
    className="a-logo-container__image"
  >
    <img
      alt="logo"
      src="/assets/logo.svg"
    />
  </div>
  <p
    className="a-logo-container__title"
  >
    MY MUSIC APP
  </p>
</div>
`;
